site/templates/buy.php

```diff
@@ -86,30 +93,37 @@
       <?php endforeach ?>
     </ul>
   </section>
-
 </article>
 
-<script src="https://cdn.paddle.com/paddle/paddle.js"></script>
 <script type="text/javascript">
-  Paddle.Setup({
-    vendor: 1129,
-  });
+function paddle_price(data) {
+    const product      = data.response.products[0];
+    const currency     = product.currency;
+    const currentPrice = product.price.net;
+    const listPrice    = product.list_price.net;
+    const isSale       = currentPrice !== listPrice;
 
-  Paddle.Product.Prices(499826, function(prices) {
-    let price = prices.price.net.replace(".00", "").replace("US$", "$");
-    let priceElement = document.querySelector(".price");
-    let vatElement = document.querySelector(".vat");
+    const formatter = (price) => {
+      return new Intl.NumberFormat("en", { style: "currency", currencyDisplay: "narrowSymbol", currency: currency, minimumFractionDigits: 0 }).format(price)
+    };
```

Comment:
I think `formatter` should just be set to the `NumberFormat` object and then the `.format()` function can be called on it below. Spawning a new formatter for each price is not needed.

Also: Careful about the `narrowSymbol` display. It's not supported in every browser, e.g. current versions of Safari.

Reply:
```js
let formatter;
try {
  formatter = new Intl.NumberFormat("en, {
    style: "currency",
    currency,
    currencyDisplay: "narrowSymbol"
  });
} catch (e) {
  if (e.constructor !== RangeError) {
    throw e;
  }

  formatter = new Intl.NumberFormat("en, {
    style: "currency",
    currency
  });
}
```

Reply:
That's unfortunate. That will give us long strings for many countries still then:

<img width="1324" alt="Screen Shot 2021-04-17 at 16 20 50" src="https://user-images.githubusercontent.com/3788865/115116266-03f2f980-9f99-11eb-93b6-64e46aae02c4.png">

<img width="1310" alt="Screen Shot 2021-04-17 at 16 21 11" src="https://user-images.githubusercontent.com/3788865/115116271-05bcbd00-9f99-11eb-9933-23dae281ddf2.png">

Reply:
GitHub only showed your first comment